engine/src/main/resources/assets/i18n/menu_fr.lang

```diff
@@ -0,0 +1,136 @@
+    "environment-effects-low": "Bas", 
+    "environment-effects-off": "Non", 
+    "exit-game": "Quitter Terasology", 
+    "eye-adaption": "Adaptation des yeux", 
+    "flickering-lights": "Lumières scintillantes", 
+    "fov-amount": "Champ de vision", 
+    "framerate-limit": "Frame Limit", 
+    "game-name": "Nom de la partie", 
+    "game-settings": "Paramètres", 
+    "game-worlds": "Mondes", 
+    "gameplay": "Modèle de jeu", 
+    "gameplay-info": "Vous pouvez cliquer sur 'modules' pour les paramètres avancés", 
+    "host-multiplayer-game": "Accueillir une partie", 
+    "ingame-menu-title": "Menu de jeu", 
+    "input-settings": "Input", 
+    "join-game-online": "Joindre la partie", 
+    "join-server-action": "Join", 
+    "join-server-title": "Rejoindre le serveur", 
+    "light-shafts": "Light Shafts", 
+    "listed-servers": "Liste", 
+    "load-game": "Charger", 
+    "missing-name-message": "You must enter a name", 
+    "module-dependencies-empty": "Ce module n'a pas de dépendances", 
+    "module-dependencies-exist": "Module dependencies", 
+    "module-list": "Modules", 
+    "module-name": "Name", 
+    "module-status": "Status", 
+    "module-version-available": "Disponible", 
+    "module-version-installed": "Installé", 
+    "motion-blur": "Motion Blur", 
+    "music-volume": "Volume de la musique", 
+    "oculus-rift-support": "Oculus Rift Support", 
+    "outline": "Outline", 
+    "parallax-mapping": "Parallax Mapping", 
+    "player-color": "Player Color", 
+    "player-name": "Nom du joueur", 
+    "player-settings": "Joueur", 
+    "player-settings-title": "Player Settings", 
+    "preview-world-title": "Previsualisation du monde...", 
+    "preview-zoom-factor": "Zoom", 
+    "remove-server": "Enlever", 
+    "reset-fov-amount": "Réinitialisation du champ de vision", 
+    "resolution-scale": "Résolution", 
+    "respawn": "Réapparaître", 
+    "return-main-menu": "Retour au menu principal", 
+    "screenshot-format": "Screenshot format", 
+    "screenshot-size": "Taille de la capture d'écran", 
+    "screenshot-size-double": "Double Size", 
+    "screenshot-size-half": "Half Size", 
+    "screenshot-size-normal": "Normal Size", 
+    "screenshot-size-quarter": "Un quart", 
+    "select-game-title": "Select Game", 
+    "select-modules": "Modules...", 
+    "select-modules-title": "Select Modules...", 
+    "select-world-gen": "Choose World Generator", 
+    "select-world-gen-info": "enabling modules may add more", 
+    "server-address": "Address", 
+    "server-name": "Nom", 
+    "server-owner": "Propriétaire", 
+    "server-port": "Port", 
+    "settings-language": "Language", 
+    "settings-title": "Paramètres", 
+    "shadows": "Ombres", 
+    "shadows-off": "Off", 
+    "shadows-on": "Oui", 
+    "shadows-pcr": "Activé (PCR)", 
+    "sound-volume": "Sound Volume", 
+    "ssao": "SSAO", 
+    "start-game": "Jouer", 
+    "start-singleplayer-game": "Singleplayer", 
+    "this-language-English": "English", 
```

Comment:
Agreed. I have your comment in the other PR on my list. Maybe this could go into `menu.lang` as values.